### PR TITLE
Python 3.12

### PR DESCRIPTION
**Pull request by thewtex**

### CODE REVIEW

File: scripts/macpython-install-python.sh [hunk 1]
```diff
@@ -405,16 +408,16 @@ function make_workon_venv {
...
 else
-  for pyversion in $LATEST_3p7; do
-    install_macpython $pyversion 10.9
-    install_virtualenv
-  done
-  for pyversion in $LATEST_3p8 $LATEST_3p9 $LATEST_3p10 $LATEST_3p11; do
+  #for pyversion in $LATEST_3p7; do
+    #install_macpython $pyversion 10.9
+    #install_virtualenv
+  #done
```
Comment (tbirdso) [inline scripts/macpython-install-python.sh:419]: Remove?

**Review by tbirdso — APPROVED** [review]
🚀

File: scripts/macpython-install-python.sh (hunk 1 above)
Comment (thewtex) [reply scripts/macpython-install-python.sh:419]: I thought it would be helpful to leave as a reminder that 3.7 is associated with 10.9.

Comment (tbirdso) [reply scripts/macpython-install-python.sh:419]: Ah, I didn't catch that. Would you please add a one-line comment to that effect?

**Review by dzenanz — APPROVED** [review]

File: scripts/macpython-install-python.sh (hunk 1 above)
Comment (jcfr) [inline scripts/macpython-install-python.sh:419]:
```suggestion
  # Deployment target requirements:
  # * 10.9: Python 3.7
  # * 11: Python >= 3.8
```

**Review by jcfr — APPROVED** [review]

File: scripts/dockcross-manylinux-set-vars.sh [hunk 2]
```diff
@@ -39,11 +39,11 @@ TARGET_ARCH=${TARGET_ARCH:=x64}
 
 # Specialized manylinux image tag to use for building.
 if [[ ${MANYLINUX_VERSION} == _2_28 && ${TARGET_ARCH} == x64 ]]; then
-  IMAGE_TAG=${IMAGE_TAG:=20230106-1aeaea0}
+  IMAGE_TAG=${IMAGE_TAG:=20230926-9eb419c}
 elif [[ ${MANYLINUX_VERSION} == _2_28 && ${TARGET_ARCH} == aarch64 ]]; then
   IMAGE_TAG=${IMAGE_TAG:=2022-11-19-1b19e81}
```
Comment (jcfr) [inline scripts/dockcross-manylinux-set-vars.sh:44]: Should this one be updated as well ?

File: scripts/dockcross-manylinux-set-vars.sh [hunk 3]
```diff
@@ -39,11 +39,11 @@ TARGET_ARCH=${TARGET_ARCH:=x64}
...
 elif [[ ${MANYLINUX_VERSION} == 2014 ]]; then
-  IMAGE_TAG=${IMAGE_TAG:=20230106-1aeaea0}
+  IMAGE_TAG=${IMAGE_TAG:=20230926-9eb419c}
```
Comment (jcfr) [inline scripts/dockcross-manylinux-set-vars.sh:46]: For future reference, the associated dockcross changes are the following: https://github.com/dockcross/dockcross/compare/1aeaea0..9eb419c

Summary:
* CMake `3.24.2` -> `3.27.5`
* Ninja `1.11.0` -> `1.11.1`
* Git `2.36.1` -> `2.42.0`
* Manylinux `2.28`
  * Update toolset from `gcc-toolset-11` to `gcc-toolset-12`
  * `pypa/manylinux_2_28_x86_64:2022-11-19-1b19e81` ->  `quay.io/pypa/manylinux_2_28_x86_64:2022-11-28-5d13db4`
* Manylinux `2014`
  * `pypa/manylinux2014_x86_64:2022-11-28-5d13db4` -> `pypa/manylinux2014_x86_64:2023-08-27-bd7ad21`

File: scripts/dockcross-manylinux-set-vars.sh (hunk 2 above)
Comment (thewtex) [reply scripts/dockcross-manylinux-set-vars.sh:44]: This one is currently separate as we work towards arm cross-compilation.

**Pull request merged**